docker-compose.yml

```diff
@@ -2,17 +2,14 @@ version: "3.8"
 
 services:
   drivernos:
-    image: resb53/drivernos:latest
+    image: jackson147/drivernos:0.2
     logging:
       driver: "json-file"
       options:
         max-size: "5m"
         max-file: "12"
-    secrets:
-      - drivernos-secret
+    environment:
+      - TOKEN=deadbeef-a7c6-46d9-8d10-deadbeef
```

Comment:
I'm assuming this isn't a sensitive token?

Reply:
Correct! Just leaving a dummy one in as an example